src/app/i18n.service.ts

```diff
@@ -55,6 +56,8 @@ export class I18nService {
       this.currentLocale_.next(this.defaultLocale);
   }
 
+  init(): void { }
```

Comment:
There's a not yet resolved issue that requires a method like this in the app.module.ts. Otherwise dependent services receive this service as undefined.